src/System.CommandLine/Help/HelpBuilder.cs

```diff
@@ -12,7 +12,7 @@ namespace System.CommandLine.Help
     /// <inheritdoc />
     public class HelpBuilder : IHelpBuilder
     {
-        private const string Indent = "  ";
+        protected const string Indent = "  ";
```

Comment:
I think this change might make it harder to stabilize the API. I'd recommend hard-coding the indent and making sure you have the right test coverage in place to detect any change if it happens.